python/lib/include/pyimage.h

```diff
@@ -120,16 +121,20 @@ void declare_image(py::module &m, std::string typestr) {
         py::init<py::array_t<T>, const std::shared_ptr<const gauss2d::CoordinateSystem>>(),
         "data"_a, "coordsys"_a = gauss2d::COORDS_DEFAULT
     )
+    .def_property_readonly("coordsys", &Class::get_coordsys_ptr_const)
+    .def_property_readonly("data", &Class::get_data)
     .def_property_readonly("n_rows", &Class::get_n_rows)
     .def_property_readonly("n_cols", &Class::get_n_cols)
-    .def_property_readonly("data", &Class::get_data)
     .def("fill", &Class::fill)
     .def("get_value", &Class::get_value)
     .def("set_value", &Class::set_value)
     .def("get_value_unchecked", &Class::get_value_unchecked)
     .def("set_value_unchecked", &Class::set_value_unchecked)
     .def_property_readonly("size", &Class::size)
     .def_property_readonly("shape", &Class::shape)
+    .def(py::self == py::self)
+    .def(py::self != py::self)
+    .def(py::self += T())
```

Comment:
I've never seen any of these before. What do they do?

Reply:
They're [various operators](https://pybind11.readthedocs.io/en/stable/advanced/classes.html#operator-overloading). The (in)equality ones are particularly strange to look at, but it is what it is.